core/src/main/resources/tokens/slds/components.json

```diff
@@ -3,7 +3,8 @@
   "annotations" : {
     "base" : "true",
     "name" : "accordion",
-    "role" : ".slds-accordion",
+    "role" : null,
+    "lwc" : "true",
```

Comment:
I wanted to double check, are these json file updates for 232 or 234?

Reply:
They are for 232.

Reply:
Ok great, thanks @sfdctaka !